src/components/WorkOrders/CloseWorkOrder.js

```diff
@@ -141,19 +147,36 @@ const CloseWorkOrder = ({ reference }) => {
     setCloseWorkOrderFormPage(!CloseWorkOrderFormPage)
   }
 
+  const updateTotalPercentage = (percentageNumber) => {
+    setTotalPercentage(percentageNumber)
+  }
```

Comment:
I'm not sure if we gain much from wrapping `setTotalPercentage` here in another function. Could we just pass `setTotalPercentage` down to the children (and name the props on them to match)? Keeping the convention makes sense to me unless we have a reason not to.